joschahenningsen/TUM-Live
# Copy & Move videos to different courses

### Motivation and Context



Lecturers should be able to move and copy lectures from previous semesters. This makes their lives easier as they don't have to reupload material and saves disk space because we don't need to store videos twice.

### Description


New interface in the admin panel allows moving lectures to any course that the user is admin of

### Steps for Testing

Prerequisites:
- 1 Lecturer
- 2 courses the lecturer is admin of
- 1 stream/vod

1. Go to admin page of course with a lecture and copy & move the lecture to another course
2. verify it has been copied or moved

### Screenshots


https://github.com/joschahenningsen/TUM-Live/assets/44805696/a53bbea0-a1e3-4481-939f-65fdff4dcc09
